beacon-chain/db/validator.go

```diff
@@ -72,9 +72,29 @@ func (db *BeaconDB) DeleteValidatorIndex(pubKey []byte) error {
 	})
 }
 
+// HasValidators checks if a validator index map exists out of a list of public keys.
+func (db *BeaconDB) HasValidators(pubKeys [][]byte) bool {
```

Comment:
You do multiple reads here for checking if there are multiple validators, this can be done in a single read.

Reply:
Thanks

Reply:
Please change the name to HasAnyValidator